src/streamdiffusion/tools/install-tensorrt.py

```diff
@@ -41,6 +41,10 @@ def install(cu: Optional[Literal["11", "12"]] = get_cuda_version_from_torch()):
         run_pip(
             "install onnx-graphsurgeon==0.3.26 --extra-index-url https://pypi.ngc.nvidia.com"
         )
+    if not is_installed("pywin32"):
```

Comment:
hi @cumulo-autumn and @ddPn08 , running this on linux and getting an error since `pywin32` only works on windows

Maybe you want to do 
```python
    if not is_installed("pywin32") and os.name == 'nt':
```